src/mixin.ts

```diff
@@ -105,6 +99,8 @@ export function mixin(Vue: VueConstructor) {
       const bindingObj = binding
       vmStateManager.set(vm, 'rawBindings', binding)
 
+      // binding = proxyRefs(binding);
+
```

Comment:
```suggestion
      // binding = proxyRefs(binding);

```

Guess it's temporary?